### PR TITLE
enable prettier in azure-local-service

### PR DESCRIPTION
**Pull request by jikim-msft**
#### Description
Board: https://dev.azure.com/fluidframework/internal/_workitems/edit/2118/

This PR will "eventually" allow running `prettier` in each sub-directory of release group and packages. Instead of adding the script to the root directory, developers can run `npm format` in whatever directory they are currently working on which will fix <b>entire files</b> accordingly to the prettier configuration. 

For now, the script is added to `azure/packages/azure-local-service` but will be applied to rest of the packages in `azure` release group.

### CODE REVIEW

File: azure/packages/azure-local-service/package-lock.json [hunk 1]
```diff
@@ -0,0 +1,7680 @@
+{
```
Comment (Josmithr) [inline azure/packages/azure-local-service/package-lock.json:1]: We don't want to add this file. The azure release group has a single set of lock files at its root. You'll want to re-run `npm install` from the root of the release group, and remove this file.

Comment (Josmithr) [reply azure/packages/azure-local-service/package-lock.json:1]: This file's addition is also why the PR pipeline is currently failing (https://dev.azure.com/fluidframework/public/_build/results?buildId=108376&view=logs&j=12f1170f-54f2-53f3-20dd-22fc7dff55f9&t=baa7d181-d9ee-513a-8016-f44a98fd040e)

**Review by Josmithr — COMMENTED** [review]
Looks great! I left one comment, but otherwise this looks ready to go.

File: azure/packages/test/scenario-runner/package.json [hunk 2]
```diff
@@ -81,6 +84,7 @@
     "fluid-framework": "^1.4.0-108057",
     "js-yaml": "^4.1.0",
     "mocha": "^10.0.0",
+    "prettier": "~2.6.2",
```
Comment (alexvy86) [inline azure/packages/test/scenario-runner/package.json:87]: Should this go in `devDependencies`?

File: azure/packages/azure-local-service/package.json [hunk 3]
```diff
@@ -23,15 +23,19 @@
...
+    "prettier": "prettier --check . --ignore-path ../../../.prettierignore",
+    "prettier:fix": "prettier --write . --ignore-path ../../../.prettierignore",
     "start": "forever start dist/index.js",
     "start:debug": "node --inspect=0.0.0.0:9229 dist/index.js",
     "stop": "forever stop dist/index.js",
     "test": "echo \"Error: no test specified\" && exit 1",
     "tsc": "tsc"
   },
   "dependencies": {
+    "prettier": "~2.6.2",
```
Comment (alexvy86) [inline azure/packages/azure-local-service/package.json:38]: Ditto

Comment (jikim-msft) [reply azure/packages/azure-local-service/package.json:38]: I added `prettier` in `azure-local-service` and `test/scenario-runner` packages. 

Rest of the other packages already had `prettier` with them. For example, I was already able to run `npm format` to run prettier in `azure-client` packages. 

Prettier is also included as a `devDependencies` inside the `azure-client/package.json`, thus I thought it was okay to apply the existing pattern to other packages within the other azure packages.

Comment (alexvy86) [reply azure/packages/azure-local-service/package.json:38]: You say that `azure-client` has it in `devDependencies`, not `dependencies`, correct? My point is to do the same here. The two packages in this PR have it under `dependencies`.

**Pull request closed without merging**